### ISSUE TITLE
Crash when starting a new world in singleplayer

### ISSUE THREAD

**Issue by PaintNinja** [thislooksfun/ForgeUpdater#1]
With this mod installed, I am unable to test my mod "Ninja's Cash" as it crashes during the loading of a singleplayer world.

I am using Forge Updater-1.7.10-1.3.1.jar and Forge #1291

Log: http://pastebin.com/SEsStf0R

**Comment by thislooksfun** [comment]
What other mods do you have installed? This is definitely a conflict.

**Comment by PaintNinja** [comment]
It's running in a clean Forge dev environment with my mod "Ninja's Cash".

I used this code to let ForgeUpdater know about updates for my mod, following the provided documentation on this Github repository:

``` java
@Mod(modid="ninjascash", name="Ninja's Cash", version="2.1")
public class NinjasCash {
    @Instance(value = "ninjascash")
    public static NinjasCash instance;

    //Update checking via ForgeUpdater mod//
    //Update checking settings
    int updateCheckType = 2;

    //If modid forgeupdater is installed, tell it that this mod's curse ID is...
    @Optional.Method(modid = "forgeupdater")
    public String curseID() {
        return "227985-ninjas-cash"; //...that
    }

    //Tell forgeupdater what types of updates it should check for. 2 = stable only. 1 = beta and stable. 0 = alpha, beta and stable.
    @Optional.Method(modid = "forgeupdater")
    public int minType() {
        return updateCheckType;
    }
```

**Comment by thislooksfun** [comment]
Ok, that's weird. The error is complaining that the method [CommandUpdates.getCommandName()](https://github.com/thislooksfun/ForgeUpdater/blob/master/ForgeUpdater/src/com/tlf/forgeupdater/command/CommandUpdates.java#L23-L27) doesn't exist. I'm trying to install a 1291 environment to see if that line is different.

Also, I completely forgot to update the [documentation](https://github.com/thislooksfun/ForgeUpdater/blob/master/Documentation.md) when I put in the new system, but it's there now. Please use that instead, as it's _much_ easier for me to maintain. If you have any questions about how it works, please ask. I might have missed something when I was transferring it over.

**Comment by thislooksfun** [comment]
Ok, I checked, and I have no idea why this isn't working. I can see no reasons why it shouldn't. O_o
Unfortunately, I can't do any more work on it now, but hopefully I'll remember to come back to it when I have time.

**Comment by PaintNinja** [comment]
I can confirm that the issue still exists after changing my mod's update code as according to your new documentation. Btw, I prefer the new system! :+1:

Let me know once you think you may have the issue sorted. :)

**Comment by thislooksfun** [comment]
Yeah, that's why I changed it over. It's also much easier to maintain, as it doesn't use reflection, unlike the other one. :P
Hopefully I'll have time later today, but we'll see.

**Comment by thislooksfun** [comment]
Ok, I just tested it myself, and I can't seem to reproduce the error...
How is your environment set up? Could I get a screenshot of it?

**Comment by PaintNinja** [comment]
My dev. environment is setup like so:
- Unzip recommended Forge 1.7.10 src to F:\Modding\NinjasCash
- Run gradlew setupDevWorkspace
- Run gradlew eclipse
- Setup eclipse workspace as F:\Modding\NinjasCash\eclipse
- -Screenie: http://oi58.tinypic.com/1llxw.jpg
- Put ForgeUpdater in F:\Modding\NinjasCash\eclipse\mods
- -Screenie:http://oi59.tinypic.com/sblqfb.jpg

**Comment by thislooksfun** [comment]
Aha! There's your problem... Hopefully. Install CodeChickenCore, and it should work just fine. Compiled mods don't work in a deobf workspace, but CCC makes it work. Don't ask me how, I think it's magic, but there you go.

**Comment by thislooksfun** [comment]
Did you figure this out?
